hw/ip/otbn/dv/uvm/get-toolchain-paths.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+# If we are on an air-gapped machine, we need to first ensure the toolchain has
+# been unpacked from the repository cache.
+if [[ -n ${BAZEL_CACHE} ]]; then
+  bazel fetch \
+    --distdir=${BAZEL_DISTDIR} \
+    --repository_cache=${BAZEL_CACHE} \
+    @com_lowrisc_toolchain_rv32imc_compiler//...
+fi
+
+# Set environment variables for the RV32 linker and assembler.
+export RV32_TOOL_LD=$(bazel query \
+  'deps(@com_lowrisc_toolchain_rv32imc_compiler//:bin/riscv32-unknown-elf-ld)' \
+  --output location | awk '{split($0,a,":"); print a[1]}')
```

Comment:
Is the `awk` command equivalent to `cut -f1 -d:`? :)

Reply:
it is, and the `cut` command is much cleaner. Thanks!

Fixed.